ResearchKit/Common/ORKImageCaptureView.m

```diff
@@ -119,6 +115,7 @@ - (void)orientationDidChange {
     }
     [_previewView setVideoOrientation:orientation];
     [self.delegate videoOrientationDidChange:orientation];
+    [self setNeedsUpdateConstraints];
```

Comment:
The removal of constraints also needs to be back to avoid duplicate ones. I will submit the convergence PR tonight.